### PR TITLE
Added $event argument to directive host event handler

### PR DESCRIPTION
**Pull request by avielfedida**

### CODE REVIEW

**Comment by avielfedida** [comment]
I signed it!

File: modules/angular2/docs/core/02_directives.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ Here is a trivial example of a tooltip decorator. The directive will log a toolt
     'text: tooltip'          |  - DOM element tooltip property should be
   ],                         |    mapped to the directive text property.
   host: {                    | List which events need to be mapped.
-    '(mouseover)': 'show()'    |  - Invoke the show() method every time
+    '(mouseover)': 'show($event)'    |  - Invoke the show() method every time
```
Comment (vicb) [inline modules/angular2/docs/core/02_directives.md]: Why would we want to pass the `$event` here ?

**Comment by avielfedida** [comment]
I can see what you mean, when I think about it there is no use for event within show() so as you say you should remove it.

**Comment by vicb** [comment]
> I can see what you mean, when I think about it there is no use for event within show() so as you say you should remove it.

What about submitting a PR ?

**Comment by avielfedida** [comment]
Sorry when you said: "I would rather remove" I was thinking you'll do it, but I'd be glad to do so, thanks.

**Pull request closed without merging**